charts/crowdsec/templates/lapi-deployment.yaml

```diff
@@ -72,12 +80,25 @@ spec:
         env:
           - name: MB_DB_FILE
             value: /metabase-data/metabase.db
-      {{ end }}
+      {{- end }}
       terminationGracePeriodSeconds: 30
-      {{ if .Values.lapi.dashboard.enabled }}
+      {{- if or (.Values.lapi.persistentVolume.data.enabled) (.Values.lapi.persistentVolume.config.enabled) (.Values.lapi.dashboard.enabled) }}
       volumes:
+      {{- if .Values.lapi.dashboard.enabled }}
       - name: shared-data
         emptyDir: {}
+      {{- end }}
+      {{- if .Values.lapi.persistentVolume.data.enabled }}
+      - name: crowdsec-db
+        persistentVolumeClaim:
+          claimName: {{ .Release.Name }}-db-pvc
+      {{- else if .Values.lapi.dashboard.enabled }}
       - name: crowdsec-db
         emptyDir: {}
-      {{ end }}
\ No newline at end of file
+      {{- end }}
+      {{- if .Values.lapi.persistentVolume.config.enabled }}
```

Comment:
This volume isn't used here, we can just avoid mounting it in this container (the metabase dashboard only needs crowdsec database)